kt/api-generator/src/main/kotlin/godot/codegen/services/impl/GenerationService.kt

```diff
@@ -150,6 +150,11 @@ class GenerationService(
             constantsTypeReceiver.addProperty(generateConstant(constant, name))
         }
 
+        for (method in enrichedClass.methods.filter { it.internal.isStatic }) {
+            //TODO/4.0: Change implementation for static method
```

Comment:
Should this comment stay here ?

Reply:
TBH i don't know. I did not add it and i don't know why it was here ^^

Reply:
IIRC, it was here to remember to do this PR